src/ui/flutter_app/pubspec.yaml

```diff
@@ -1,44 +1,45 @@
 name: sanmill
 description: Sanmill is a open-source, powerful UCI-like Nine Men's Morris (and its variants) program.
+publish_to: none
 
 version: 1.1.37+2184
 
 environment:
   sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
-  flutter:
-    sdk: flutter
-  flutter_localizations:
-    sdk: flutter
-  cupertino_icons: ^1.0.3
-  soundpool: ^2.2.0
-  path_provider: ^2.0.5
-  package_info_plus: ^1.0.6
-  uuid: ^3.0.4
-  url_launcher: ^6.0.11
-  intl: 0.17.0
+  animated_text_kit: ^4.1.1
   catcher: ^0.6.8
-  stack_trace: ^1.10.0
-
+  cupertino_icons: ^1.0.3
+  device_info_plus_platform_interface: ^2.1.0
   devicelocale: ^0.4.3
   double_back_to_close_app: ^2.0.1
-  flutter_picker: ^2.0.2
-  flutter_email_sender: ^5.0.2
   feedback:
     git:
       url: git://github.com/calcitem/feedback.git
-  animated_text_kit: ^4.1.1
-  flutter_colorpicker: ^0.6.0
-  device_info_plus_platform_interface: ^2.1.0
   fluentui_system_icons: ^1.1.140
+  flutter:
+    sdk: flutter
+  flutter_colorpicker: ^0.6.0
+  flutter_email_sender: ^5.0.2
+  flutter_localizations:
+    sdk: flutter
+  flutter_picker: ^2.0.2
+  intl: 0.17.0
+  package_info_plus: ^1.0.6
+  path_provider: ^2.0.5
+  soundpool: ^2.2.0
+  stack_trace: ^1.10.0
+  url_launcher: ^6.0.11
+  uuid: ^3.0.4
   #pref: ^2.3.0
   #screen_recorder: ^0.0.2
 
 dev_dependencies:
   flutter_oss_licenses: ^1.0.1
   flutter_test:
     sdk: flutter
+  lint: ^1.0.0
```

Comment:
Why are you not using the latest version, v1.7.2?

Reply:
My bad
I thought I was using It